datacontract/model/data_contract_specification.py

```diff
@@ -186,6 +186,8 @@ class Model(pyd.BaseModel):
     title: Optional[str] = None
     fields: Dict[str, Field] = {}
     quality: List[Quality] | None = []
+    primaryKey: List[str] | None = []
+    examples: List[Any] | None = None
```

Comment:
I made it the same type as https://github.com/datacontract/datacontract-cli/blob/ff6ec618683f54ee1dcf16223e541fdfb4e00995/datacontract/model/data_contract_specification.py#L173